### ISSUE TITLE
Not binding correctly steps using examples from Scenario Outline

### ISSUE THREAD

**Issue by vsrelvas** [SpecFlowOSS/SpecFlow.Rider#18]
Cannot navigate to the step definitions which are referencing headers in the examples table, in a scenario outline like this:

```
Scenario Outline: eating
  Given there are <start> cucumbers
  When I eat <eat> cucumbers
  Then I should have <left> cucumbers

  Examples:
    | start | eat | left |
    |    12 |   5 |    7 |
    |    20 |   5 |   15 |
```

**Comment by Socolin** [comment]
This should be fixed in 1.3.3. It should be available soon on the marketplace.
If you want to test it now, it's available here https://github.com/SpecFlowOSS/resharper-specflow/releases/tag/v1.3.3

**Comment by nemesv** [comment]
It is working in version in 1.3.3
